draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -1218,568 +1049,686 @@ uCDN-A                         dCDN-B                         dCDN-C
-
-
-   <section title="CI/T Modify Command" anchor="cit-trigger-modify-command">
-   <t> 
-   CI/T Modify Command, which is used to modify an existing trigger, MUST use a MIME media 
-   type of "application/cdni; ptype=ci-trigger-command.modify.v2".
+   Mandatory: Yes. Furthermore, the list MUST NOT be empty.
    </t>
-   <t>
-      A CI/T Modify Trigger Command is encoded as a JSON object containing the following
-      name/value pairs:</t>
 
-   <t><list style="empty" hangIndent="3">
-   <t><list style="hanging" hangIndent="3"><t hangText="Name: specs">
+   
+   <t hangText="Name: extensions">
    <vspace blankLines="1"/>
-   Description: An updated specification of the trigger action and a set of targets upon which to act.
+   Description: Array of trigger extensions, as described in <xref target="trigger-extensions"/>.
    <vspace blankLines="1"/>
-   Value: A Trigger.v2 Specification, as defined in <xref target="trigger-v2-specification"/>.
+   Value: Array of GenericTriggerExtension objects (see <xref target="generic-extension-object"/>).
    <vspace blankLines="1"/>
-   Mandatory: No, but at least one of "specs" and "desired-status" should be set.
+   Mandatory: No. The default is no extensions.
    </t>
 
-   <t hangText="Name: desired-status">
-   <vspace blankLines="1"/>
-   Description: The target status for an existing trigger as defined in <xref target="trigger-status"/>
+   <t hangText="Name: labels">
    <vspace blankLines="1"/>
-   Value: JSON string. 
+   Description: Array of trigger labels.
    <vspace blankLines="1"/>
-   Mandatory: No, but at least one of "specs" and "desired-status" should be set.
-   </t>
-
-   </list></t>
-   </list></t>
-
-   </section>
-
+   Value: Array of Trigger labels, Each label is encoded as a JSON string and SHOULD be unique per trigger.
 
-   <section title="Trigger Status Resources" anchor="trigger-status-resources"><t>
-    Trigger Status Resources MUST use a MIME media type of
-   "application/cdni; ptype=ci-trigger-status.v2".</t>
 
-   <t>
-   A Trigger Status Resource is encoded as a JSON object containing the
-   following name/value pairs:</t>
+   <vspace blankLines="1"/>
+   Mandatory: No. The default is no labels.
+   </t>
 
-   <t><list style="empty" hangIndent="3">
-   <t><list style="hanging" hangIndent="3"><t hangText="Name: trigger">
+   <t hangText="Name: cdn-path">
    <vspace blankLines="1"/>
-   Description: The Trigger Specification POSTed in the body of
-         the CI/T Command. Note that this need not be a byte-for-byte
-         copy. For example, in the JSON representation dCDN may
-         re-serialize the information differently.
+   Description: The chain of CDN PIDs of CDNs that have already created this trigger resource. 
    <vspace blankLines="1"/>
-   Value: A Trigger.v2 Specification, as defined in <xref target="trigger-v2-specification"/>.
+   Value: Non-empty array of JSON strings, where each string is a CDN PID as defined in <xref target="cdn-pid"/>.
```

Comment:
So, since we now have a dedicated section on the CDN PID, this appears to be the only place that references it.  Other references are still resolving to "loop-detection-and-prevention".  Maybe we should adjust these for consistency?

Reply:
fixed